resources/i18n/zh_TW/uranium.po

```diff
@@ -0,0 +1,868 @@
+
+#: /home/ruben/Projects/Uranium/plugins/Tools/RotateTool/__init__.py:13
+msgctxt "@info:tooltip"
+msgid "Rotate Model"
+msgstr "旋轉模型"
+
+#: /home/ruben/Projects/Uranium/plugins/Tools/MirrorTool/__init__.py:12
+msgctxt "@label"
+msgid "Mirror"
+msgstr "鏡像"
+
+#: /home/ruben/Projects/Uranium/plugins/Tools/MirrorTool/__init__.py:13
+msgctxt "@info:tooltip"
+msgid "Mirror Model"
+msgstr "鏡像模型"
+
+#: /home/ruben/Projects/Uranium/UM/Settings/Models/InstanceContainersModel.py:168
+#: /home/ruben/Projects/Uranium/UM/Qt/Bindings/MeshFileHandlerProxy.py:48
+#: /home/ruben/Projects/Uranium/UM/PluginRegistry.py:77
+#: /home/ruben/Projects/Uranium/UM/FileHandler/FileHandler.py:70
+#, python-brace-format
+msgctxt "@item:inlistbox"
+msgid "All Supported Types ({0})"
+msgstr "所有支援的檔案類型 ({0})"
+
+#: /home/ruben/Projects/Uranium/UM/Settings/Models/InstanceContainersModel.py:171
+#: /home/ruben/Projects/Uranium/UM/Qt/Bindings/MeshFileHandlerProxy.py:49
+#: /home/ruben/Projects/Uranium/UM/PluginRegistry.py:78
+#: /home/ruben/Projects/Uranium/UM/FileHandler/FileHandler.py:71
+msgctxt "@item:inlistbox"
+msgid "All Files (*)"
+msgstr "所有檔案 (*)"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:108
+#, python-brace-format
+msgctxt ""
+"@label Short days-hours-minutes format. {0} is days, {1} is hours, {2} is "
+"minutes"
+msgid "{0:0>2}d {1:0>2}h {2:0>2}min"
+msgstr "{0:0>2}天 {1:0>2}時 {2:0>2}分"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:110
+#, python-brace-format
+msgctxt "@label Short hours-minutes format. {0} is hours, {1} is minutes"
+msgid "{0:0>2}h {1:0>2}min"
+msgstr "{0:0>2}時 {1:0>2}分"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:113
+#, python-brace-format
+msgctxt "@label Long duration format. {0} is days"
+msgid "{0} day"
+msgid_plural "{0} days"
+msgstr[0] "{0} 天"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:113
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:115
+#, python-brace-format
+msgctxt "@label Long duration format. {0} is hours"
+msgid "{0} hour"
+msgid_plural "{0} hours"
+msgstr[0] "{0} 時"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:113
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:115
+#: /home/ruben/Projects/Uranium/UM/Qt/Duration.py:117
+#, python-brace-format
+msgctxt "@label Long duration format. {0} is minutes"
+msgid "{0} minute"
+msgid_plural "{0} minutes"
+msgstr[0] "{0} 分"
+
+#: /home/ruben/Projects/Uranium/UM/Qt/QtApplication.py:115
+msgctxt "@info:progress"
+msgid "Loading plugins..."
+msgstr "正在載入外掛..."
+
+#: /home/ruben/Projects/Uranium/UM/Qt/QtApplication.py:120
+msgctxt "@info:progress"
+msgid "Updating configuration..."
+msgstr "正在更新設定..."
+
+#: /home/ruben/Projects/Uranium/UM/Qt/QtApplication.py:131
+msgctxt "@info:progress"
+msgid "Loading preferences..."
+msgstr "正在載入偏好設定..."
+
+#: /home/ruben/Projects/Uranium/UM/Mesh/ReadMeshJob.py:67
+#, python-format, python-brace-format
+msgctxt "@info:status"
+msgid "Auto scaled object to {0}% of original size"
+msgstr "自動將物件縮放到原始大小的 {0}% o"
```

Comment:
Seems like there's a little mistake here?

Reply:
The 'o' was put at the end of string intentionally. I use Poedit 2.0.4 to convert po file to mo file. If I removed the 'o', poedit pop up a error message "msgstr is not a valid Python format string."

Reply:
PoEdit might be mistaken there then. It is indeed not a valid Python format string, but it's not actually formatted in that way so it's a bit of a weird constraint to put on your texts... Oh well, next time maybe.